src/apis/stockx/user/helpers.js

```diff
@@ -0,0 +1,131 @@
+import cheerio from 'cheerio';
+
+export const getState = async ({ request, headers, jar, proxy }) => {
+  try {
+    const res = await request('https://stockx.com/login', {
+      followAllRedirects: true,
+      followRedirect: true,  
+      headers: {
+        ...headers,
+        accept: 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3',
+        'sec-fetch-mode': 'navigate',
+        'sec-fetch-site': 'none',
+        'sec-fetch-user': '?1',
+      },
+      jar,
+      proxy,
+      resolveWithFullResponse: true
+    });
+
+    const { statusCode, req: { _header } } = res;
+
+    if (!statusCode || (statusCode && statusCode !== 200)) {
+      const err = new Error('Invalid response code!');
+      err.status = statusCode || 404;
+      throw err;
+    }
```

Comment:
This is duplicated quite a bit, refactoring it to a common function would be cleaner.